src/data/restore.ts

```diff
@@ -13,7 +13,9 @@ export function restore(
   opts?: { scrollToContent: boolean },
 ): DataState {
   const elements = savedElements
-    .filter(el => !isInvisiblySmallElement(el))
+    .filter(el => {
+      return el.type !== "selection" && !isInvisiblySmallElement(el);
```

Comment:
Can you add a comment for selection saying that this used to be a bug and should no longer be saved. So this is for backwards compatibility.